com.visualstudio.code.yaml

```diff
@@ -13,6 +13,7 @@ finish-args:
   - --share=network
   - --share=ipc
   - --socket=x11
```

Comment:
Would it not be good to use `--socket=fallback-x11` here instead?
(https://docs.flatpak.org/en/latest/sandbox-permissions.html)

Reply:
I agree, I've changed that permission